package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "fastify-secrets-hashicorp",
+  "version": "0.0.1",
+  "description": "Fastify secrets plugin for HashiCorp Vault",
+  "main": "lib/fastify-secrets-hashicorp.js",
+  "scripts": {
+    "lint": "eslint lib/**/*.js test/**/*.js",
+    "lint:fix": "npm run lint -- --fix",
+    "lint:staged": "lint-staged",
+    "test": "tap --reporter=spec --coverage-report=html --coverage-report=text --100 --no-browser test/*.test.js test/**/*.test.js",
+    "test:ci": "tap --no-color --reporter=spec --coverage-report=json --coverage-report=text --100 test/*.test.js test/**/*.test.js"
```

I believe you can simply do:

```suggestion
    "test": "tap --reporter=spec --coverage-report=html --coverage-report=text --100 --no-browser test",
    "test:ci": "tap --no-color --reporter=spec --coverage-report=json --coverage-report=text --100 test"
```